server/data/data-postgres/src/test/java/org/apache/james/sieve/postgres/PostgresSieveRepositoryTest.java

```diff
@@ -17,30 +17,39 @@
  * under the License.                                           *
  ****************************************************************/
 
-package org.apache.james.sieve.jpa;
+package org.apache.james.sieve.postgres;
 
 import org.apache.james.backends.jpa.JpaTestCluster;
-import org.apache.james.sieve.jpa.model.JPASieveQuota;
-import org.apache.james.sieve.jpa.model.JPASieveScript;
+import org.apache.james.backends.postgres.PostgresExtension;
+import org.apache.james.backends.postgres.PostgresModule;
+import org.apache.james.backends.postgres.quota.PostgresQuotaCurrentValueDAO;
+import org.apache.james.backends.postgres.quota.PostgresQuotaLimitDAO;
+import org.apache.james.backends.postgres.quota.PostgresQuotaModule;
+import org.apache.james.sieve.postgres.model.JPASieveScript;
 import org.apache.james.sieverepository.api.SieveRepository;
 import org.apache.james.sieverepository.lib.SieveRepositoryContract;
 import org.junit.jupiter.api.AfterEach;
 import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.RegisterExtension;
 
-class JpaSieveRepositoryTest implements SieveRepositoryContract {
+class PostgresSieveRepositoryTest implements SieveRepositoryContract {
+    @RegisterExtension
+    static PostgresExtension postgresExtension = PostgresExtension.withoutRowLevelSecurity(PostgresModule.aggregateModules(PostgresQuotaModule.MODULE));
 
-    final JpaTestCluster JPA_TEST_CLUSTER = JpaTestCluster.create(JPASieveScript.class, JPASieveQuota.class);
+    final JpaTestCluster JPA_TEST_CLUSTER = JpaTestCluster.create(JPASieveScript.class);
 
     SieveRepository sieveRepository;
 
     @BeforeEach
     void setUp() {
-        sieveRepository = new JPASieveRepository(JPA_TEST_CLUSTER.getEntityManagerFactory());
+        sieveRepository = new PostgresSieveRepository(JPA_TEST_CLUSTER.getEntityManagerFactory(),
+            new PostgresSieveQuotaDAO(new PostgresQuotaCurrentValueDAO(postgresExtension.getPostgresExecutor()),
+                new PostgresQuotaLimitDAO(postgresExtension.getPostgresExecutor())));
     }
 
     @AfterEach
     void tearDown() {
-        JPA_TEST_CLUSTER.clear("JAMES_SIEVE_SCRIPT", "JAMES_SIEVE_QUOTA");
+        JPA_TEST_CLUSTER.clear("JAMES_SIEVE_SCRIPT");
```

Comment:
Not needed, right?

Reply:
Sieve script migration: other task => https://github.com/linagora/james-project/issues/4928